telemetry-interfaces/src/main/java/scala/meta/internal/telemetry/UnknownProducerContext.java

```diff
@@ -0,0 +1,65 @@
+package scala.meta.internal.telemetry;
+
+import java.util.Map;
+
+public class UnknownProducerContext implements ReporterContext {
```

Comment:
Why do we need this? It has no data, so could probably be removed?

Reply:
I was added so we can add some arbitrary producer which is neither metals or presentaion compiler. In such case we'd use a map[string,string] to pass additional context, however, I agree that it is redundant. It was removed